### DISCUSSION TITLE
Cause/Significance of sudden jump in vibrational free energy

### DISCUSSION THREAD

**Discussion by apandit2021** [ttadano/alamode#83] [Q&A]
Dear Dr. Tadano,

Within ALAMODE implementation, I am getting strange jump in vibrational free energy curve around a finite temperature (NOT the transition temperature) for my system. What could be the physical/computational cause and/or significance of this jump?

Best,
Abhiyan

**Comment by ttadano** [comment]
Please explain more details about the jump you observed. How did you get it?

**Comment by apandit2021** [comment]
Dear Dr. Tadano,

I am trying to calculate and plot the free-energy difference -del_F (T) between two different structural phases of a 2D system, using SCP approach following your article J. Cer. Soc. of Jpn. 127 [6] 404-408 2019 as a reference (Fig. 3 there). For my system here instead of getting a smooth curve, I got a sudden jump at a certain temperature, which is much below the experimentally observed transition temperature for the system. The curve re-gains its smoothness after the jump. So, I want to get your feedback whether the jump has any physical significance/reasons and/or just computational issues. 

Best,
Abhiyan
